editor/inspector/assets/fbx/animation.js

```diff
@@ -470,6 +469,9 @@ const Elements = {
                     }
                     panel.clipNames.delete(panel.currentClipInfo.name);
                     panel.animationInfos[panel.rawClipIndex].splits.splice(panel.splitClipIndex, 1);
+                    if (panel.animationInfos[panel.rawClipIndex].splits.length > 0 && panel.splitClipIndex > 0) {
+                        panel.splitClipIndex--;
+                    }
```

Comment:
it is uncorrect logic, when delete some index item in array, 'splice' method will minus one length of  the array, so the new selection would be next item;
so the splitClipIndex is unnecessary minus.